dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-email/src/main/java/org/apache/dolphinscheduler/plugin/alert/email/EmailAlertChannel.java

```diff
@@ -38,6 +46,20 @@ public AlertResult process(AlertInfo info) {
         if (null == paramsMap) {
             return new AlertResult("false", "mail params is null");
         }
+        // Converts an object type to an array type to prevent subsequent conversions from reporting errors
+        if (StringUtils.isNotEmpty(alert.getContent())) {
```

Comment:
@jackfanwan why not just handle before `getTextTypeMessage`? Because I found the content of the Table type is defferent, see `getTableTypeMessage`.

Reply:
> @jackfanwan why not just handle before `getTextTypeMessage`? Because I found the content of the Email Table type is defferent, see `getTableTypeMessage`.

Because I see that the `toList` in the `getTableTypeMessage` method, if it's an object, it won't parse

Reply:
> @jackfanwan why not just handle before `getTextTypeMessage`? Because I found the content of the Email Table type is defferent, see `getTableTypeMessage`.

Excuse me, do I still need to make changes?

Reply:
> > @jackfanwan why not just handle before `getTextTypeMessage`? Because I found the content of the Email Table type is defferent, see `getTableTypeMessage`.
> 
> Excuse me, do I still need to make changes?

Sorry for my late, I think it's need and you had updated it.